config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -108,7 +108,7 @@ presubmits:
         - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250227-3a13bdd784-master
           env:
             - name: E2E_KIND_VERSION
-              value: kindest/node:v1.29.4
+              value: kindest/node:v1.29.14
```

Comment:
FWIW I strongly recommend controlling this sort of thing in the repo-under-test, it makes updates safer since you can test them before merge. It can also be hard for developers to replicate locally when key inputs are hidden in CI environment variables.

You could have a mapping of Kubernetes minor version to the image(s) you're using. I'd then explicitly pass whatever remaining values on the make command line so it shows up in the CI logs.

Reply:
Thank you for the recommendation, makes sense. Opened: https://github.com/kubernetes-sigs/kueue/issues/4729